src/js/view/view.js

```diff
@@ -482,23 +482,32 @@ function View(_api, _model) {
 
     function clickHandlerHelper(api, model, videoLayer) {
         const clickHandler = new ClickHandler(model, videoLayer, { useHover: true });
+        const controls = _model.get('controls');
         clickHandler.on({
             click: () => {
                 _this.trigger(DISPLAY_CLICK);
-                if (_model.get('controls')) {
-                    api.play(reasonInteraction());
+
+                if (_controls) {
+                    if (settingsMenuVisible()) {
```

Comment:
This would be a change to expected behavior. We can close the menu if it's open, similar to what you do for a tap, and always call `api.play(reasonInteraction());`.